src/Component/FormField/JSONEditor/JSONEditor.tsx

```diff
@@ -1,29 +1,23 @@
 import React, { useEffect } from 'react';
-import {
-  IControlledCodeMirror as CodeMirrorProps,
-  Controlled as CodeMirror
-} from 'react-codemirror2';
-import 'codemirror/lib/codemirror.css';
-import 'codemirror/mode/javascript/javascript';
+import Editor from '@monaco-editor/react';
 import Logger from 'js-logger';
 
 import './JSONEditor.less';
 
 type OwnProps = {
   value?: string;
-  onChange?: (value: string) =>  void;
+  onChange?: (value: string) => void;
 };
 
-export type JSONEditorProps = OwnProps & Omit<CodeMirrorProps, 'onBeforeChange' | 'value' | 'onChange'>;
```

Comment:
Non blocking, but we should allow passing the `EditorProps` to the wrapped component.

Reply:
Done in f52be66